pkg/registry/types.go

```diff
@@ -14,6 +14,24 @@ var (
 	ErrPackageNotInDatabase = errors.New("Package not in database")
 )
 
+// BundleAlreadyAddedError is an error that describes a bundle is already added
```

Comment:
What is the semantic difference between these two error types? Is there value in differentiating between that exact bundle was already added vs that specific csv was already added? Or is this doing something more subtle than what I'm reading?

Either way, the comments/naming here is confusing.

Reply:
the difference is

A bundle with a different pull spec providing the same package and csv in the same channels.

vs

The exact same bundle pull spec providing the same things.

WDYT of `PackageVersionAlreadyAdded` for the first case and `BundleAlreadyAdded` for the second?

Reply:
I like `PackageVersionAlreadyAdded`, but for the second what about `BundleImageAlreadyAdded`?

Reply:
+1 on BundleImageAlreadyAdded changes coming

Reply:
Adding `Err` on them as well